apis/frontend/oss/web-components/README.md

```diff
@@ -0,0 +1,369 @@
+# Web Components
+
+Nosto web components provide the necessary APIs to handle side-effects of a recommendation template like "Add to cart" button events, and other platform-specific APIs.
+
+**Note**:
+This package doesn't render HTML markups on its own and the template should be provided by the user.
+
+## Components
+
+This package provides the following web components:
+
+| Component         | Status            |
+| ----------------- | ----------------- |
+| `NostoProduct`      | PRODUCTION        |
+| `NostoProductCard`  | ALPHA             |
+| `NostoSkuOptions`   | PRODUCTION        |
+| `NostoShopify`      | ALPHA             |
+| `NostoSwiper`       | BETA              |
```

Comment:
Should we instead make the component names link to the corresponding section that explains the component?

Reply:
we can, but I am not sure how

Reply:
Maybe I can add it directly from gitbook, if that's okay